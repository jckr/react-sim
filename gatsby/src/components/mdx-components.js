import React from 'react';
import Prism from '@theme-ui/prism';

export default {
  pre: props => {
    return props.children;
  },
  code: props => {
    return <Prism {...props} />;
  },
};
