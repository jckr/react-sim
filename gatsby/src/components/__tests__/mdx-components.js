import React from 'react';
import renderer from 'react-test-renderer';
import { pre, code } from '../mdx-components';
describe('MDX components', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <pre>
          <code className="jsx">
            {`const MyModel = () =>
          <Model />`}
          </code>
        </pre>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
