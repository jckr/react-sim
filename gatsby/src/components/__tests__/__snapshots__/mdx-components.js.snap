// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MDX components renders correctly 1`] = `
<pre>
  <code
    className="jsx"
  >
    const MyModel = () =&gt;
          &lt;Model /&gt;
  </code>
</pre>
`;
